apps/cli/src/commands/doctor.ts

```diff
@@ -0,0 +1,47 @@
+import { Command, Flags } from '@oclif/core';
+import { execSync } from 'child_process';
+
+export default class DoctorCommand extends Command {
+  static description = 'Verify the minimal requirements for the sunodo execution commands';
+
+  static examples = ["<%= config.bin %> <%= command.id %>"];
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+  };
+
+  public async run() {
+    try {
+      // Check Docker version
+      this.log('Checking Docker installation...');
+      const dockerVersionOutput = execSync('docker -v').toString().trim();
+      const dockerVersion = dockerVersionOutput.match(/(\d+\.\d+\.\d+)/)?.[0] || 'Unknown';
+      this.log(`Docker version ${dockerVersion} is installed`);
+
+      // Check Docker Compose version
+      this.log('Checking Docker Compose installation...');
+      const dockerComposeVersionOutput = execSync('docker-compose -v').toString().trim();
+      const dockerComposeVersion = dockerComposeVersionOutput.match(/(\d+\.\d+\.\d+)/)?.[0] || 'Unknown';
+      this.log(`Docker Compose version ${dockerComposeVersion} is installed`);
+
+      // Check Docker Buildx version
+      this.log('Checking Docker Buildx installation...');
+      const dockerBuildxVersionOutput = execSync('docker buildx version').toString().trim();
+      const dockerBuildxVersion = dockerBuildxVersionOutput.match(/(\d+\.\d+\.\d+)/)?.[0] || 'Unknown';
+      this.log(`Docker Buildx version ${dockerBuildxVersion} is installed`);
+
+      // Check Docker riscv64 support
+      this.log('Checking Docker riscv64 support...');
```

Comment:
`docker buildx ls` output was important, it shows `riscv64` support, but it's kind of ugly to parse that.